AwfulSettings/Sources/AwfulSettings/Migration.swift

```diff
@@ -95,6 +96,29 @@ extension SettingsMigration {
             defaults.removeObject(forKey: oldKey)
         }
     }
+    
+    /// Migrate all users to PostImages.org to encourage its use
+    static func migrateImageHostingProvider(_ defaults: UserDefaults) {
+        // Check if user has already been migrated
+        let migrationKey = "did_migrate_to_postimages_v1"
+        if defaults.bool(forKey: migrationKey) {
+            return
+        }
+        
+        // Check current Imgur mode to handle the "Off" case
+        let imgurMode = defaults.string(forKey: Settings.imgurUploadMode.key)
+        
+        // If they had "Off", set Imgur mode to anonymous for when they switch back
+        if imgurMode == "Off" {
+            defaults.set("Anonymous", forKey: Settings.imgurUploadMode.key)
+        }
+        
+        // Move everyone to PostImages.org by default (they can switch back if they want)
+        defaults.set("PostImages", forKey: Settings.imageHostingProvider.key)
+        
```

Comment:
thinking more about this. I don’t think we should forcibly move people over if they’ve manually chosen imgur. I do like changing the default for when nothing has been chosen.

if we still want to give a little push for people who intentionally chose imgur, maybe a one time alert somewhere suggesting the change and offering to make it?